Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:18
```

Comment:
consider using the latest Node.js version and Alpine image for Docker to enhance performance and minimize container size

Reply:
Yes sir, tomorrow I'll do that